### DISCUSSION TITLE
Full text search should include option to show notes under tags that match the keyword

### DISCUSSION THREAD

**Discussion by mlncn** [standardnotes/forum#2508] [Suggestions] [answered]
That is, if i have a note titled "A People's History of the United States" with content "Excellent book; kept in our social justice library" and tagged "USA"— when i search for "USA" i expect this note to come up even though "USA" is not in the title nor the content, just the tag.

(Not sure if #64 was ever interpreted this way, but in any case this does not currently work on the hosted version at standardnotes.org.)

**Comment by atmoio** [comment]
This is interesting. I'm trying to understand why this would be desired, generally. Why not just select the USA tag and search in there? Can you give some other examples where this would be useful?

**Comment by mlncn** [comment]
Because i have to remember what i tag it.

Basically, i think that tagging by keyword and searching by keyword are very common use cases.  But no one should have to remember if the keyword was a formal tag or just in the title or text.

**Comment by vitormv** [comment]
i agree with @mobitar . Makes a lot of sense to search also in the tags as well as note title and body.

**Comment by dig12345** [comment]
Would really like to see this implemented, either as part of the main search box or a separate search box. I have lots of nested folders and its a real pain to find the correct folder when looking for notes without search.

**Comment by myreli** [comment, marked as answer]
Search now includes tags:

<img width="1257" alt="image" src="https://user-images.githubusercontent.com/17554234/218587203-2a0c6e94-1f37-494a-9053-65fa2fab50b1.png">

**Comment by dig12345** [comment]
While this is a big improvement, if there was a way to click on the tag and have the tree expanded to that node that would make this much more useful. My goal is to be able to search for a location on the tree and browse from there. Instead this returns a large collection of all notes under that node in the tree. The difference is that if I search for a folder that has lots of subfolders all the notes in the subfolders are returned in the search vs allowing me to easily browse the tree from that node and select the subfolder I wish to browse.
